src/WordPress/WordPressClientAPI.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace CF\WordPress;
+
+use CF\API\Client;
+use CF\API\Request;
+
+class WordPressClientAPI extends Client
+{
+    /**
+     * @param $zone_name
+     *
+     * @return mixed
+     */
+    public function getZoneTag($zone_name)
+    {
+        $request = new Request('GET', 'zones/', array('name' => $zone_name), array());
+        $response = self::callAPI($request);
+
+        $zone_tag = null;
+        if (self::responseOk($response)) {
+            foreach ($response['result'] as $zone) {
+                if ($zone['name'] === $zone_name) {
+                    $zone_tag = $zone['id'];
+                    break;
+                }
+            }
+        }
+
+        return $zone_tag;
+    }
+
+    /**
+     * @param $zone_tag 
+     * 
+     * @return bool
+     */
+    public function zonePurgeCache($zone_tag)
+    {
+        $request = new Request('DELETE', 'zones/'.$zone_tag.'/purge_cache', array(), array('purge_everything' => true));
+        $response = self::callAPI($request);
+
+        error_log('PURGE');
+        error_log(json_encode($response));
```

Comment:
WordPress doesn't have log levels so lets remove this since they have no way to turn it down/off.